covid19_sfbayarea/data/solano.py

```diff
@@ -220,21 +220,21 @@ def get_age_table(out: Dict) -> None:
     response2.raise_for_status()
     parsed2 = response2.json()
 
-    # Format the results.
-    age_group_cases = [{
-                        'group': entry['attributes']['Age_group'],
-                        'raw_count': entry['attributes']['AG_Total_cases']
-                       }
-                       for entry in parsed2['features']]
-    age_group_deaths = [{
-                         'group': entry['attributes']['Age_group'],
-                         'raw_count': entry['attributes']['AG_deaths']
-                        }
-                        for entry in parsed2['features']]
+    # Because of the way data is structured (ages, race/ethnicity groups, etc.
+    # all on one line), some categories get repeated (even though the data is
+    # not different). To handle this, create a dict with our groupings to
+    # de-duplicate before formatting final results.
+    age_groups = {entry['attributes']['Age_group']: entry['attributes']
+                  for entry in parsed2['features']}
```

Comment:
Just making sure I'm understanding the logic correctly: this will loop over each dict in `features`, and reassign the value for a given key (`Age_group`) if a duplicate occurs, right?

Reply:
Yeah. Comprehensions can sometimes be tough to read, so I try not to use them if they get too complex. (Have I done so here?) This is effectively equivalent to:

```py
age_groups = {}
for entry in parsed2['features']:
    key = entry['attributes']['Age_group']
    value = entry['attributes']
    age_groups[key] = value
```

So if you have multiple dicts with the same age group in `features`, the last one will win, since we are making the age group into the key for a new dictionary and then just writing over the same key again and again.

Reply:
Got it! It's clear -- just wanted to make sure I was reading it right!